src/test/java/org/apache/commons/io/DirectoryUtilsTestCase_Nested.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.commons.io;
+
+import static org.junit.Assert.*;
```

Comment:
Do not use `*` imports.

Reply:
Also based your PR on the HEAD of master. This does not compile since we now use JUnit 5.